### ISSUE TITLE
`amplify init` issues after updating to latest amplify version

### ISSUE THREAD

**Issue by alexlao** [aws-amplify/amplify-cli#5373]
**Describe the bug**
I cloned a fresh version of my current project, however upon amplify init (after updating to the latest amplify version), I am getting issues with file not found errors. I assume the issue is that a file is not being generated upon build step. 

**Amplify CLI Version**
You can use `amplify -v` to check the amplify cli version on your system
4.29.2
**To Reproduce**
Steps to reproduce the behavior or terminal output if applicable
Running amplify init 
**Expected behavior**
Amplify should be initialized such that basic functionalities should work.

**LOGS**
$ amplify init                                                                                                                                                                                          
Note: It is recommended to run this command from the root of your app directory                                                                                                                                                                
? Do you want to use an existing environment? Yes                                                                                                                                                                                              
? Choose the environment you would like to use: prod                                                                                                                                                                                           
? Choose your default editor: Vim (via Terminal, Mac OS only)
Using default provider  awscloudformation

For more information on AWS Profiles, see:
https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html

? Do you want to use an AWS profile? Yes
? Please choose the profile you want to use default
```
✖ There was an error initializing your environment.
init failed
Error: ENOENT: no such file or directory, stat '/XXX/XXXX/PROJECT/amplify/.temp/#current-cloud-backend/function/createPostAndTimeline/src/node_modules/.bin/loose-envify'
    at Object.statSync (fs.js:1016:3)
    at Object.statSync (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/graceful-fs/polyfills.js:307:34)
    at getStatsSync (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/fs-extra/lib/util/stat.js:58:18)
    at Object.checkPathsSync (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/fs-extra/lib/util/stat.js:90:33)
    at copyDirItem (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/fs-extra/lib/copy-sync/copy-sync.js:119:29)
    at /usr/local/lib/node_modules/@aws-amplify/cli/node_modules/fs-extra/lib/copy-sync/copy-sync.js:113:39
    at Array.forEach (<anonymous>)
    at copyDir (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/fs-extra/lib/copy-sync/copy-sync.js:113:23)
    at mkDirAndCopy (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/fs-extra/lib/copy-sync/copy-sync.js:108:3)
    at onDir (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/fs-extra/lib/copy-sync/copy-sync.js:99:25) {
  errno: -2,
  syscall: 'stat',
  code: 'ENOENT',
  path: '/XXX/XXXX/socialmarket/amplify/.temp/#current-cloud-backend/function/createPostAndTimeline/src/node_modules/.bin/loose-envify'
}
```
**Desktop (please complete the following information):**
 - OS: MAC
 - Node Version. You can use `node -v` to check the node version on your system
v12.18.3

**Additional context**
I was following the Boyaki guide here: https://amplify-sns.workshop.aws/en/00_prerequisites.html
After completing #5, I tried to clone my repo from another machine. However, amplify init is giving me errors with configuring my workspace.

**Comment by ammarkarachi** [comment]
@alexlao Thanks for the issue I am can you share the package.json from your lambda?

**Comment by alexlao** [comment]
Hi @ammarkarachi, thanks for helping us take a look! Here's the package.json for the lambda

{
  "name": "createPostAndTimeline",
  "version": "2.0.0",
  "description": "Lambda function generated by Amplify",
  "main": "index.js",
  "license": "Apache-2.0",
  "dependencies": {
    "aws-appsync": "^4.0.1",
    "graphql-tag": "^2.11.0",
    "node-fetch": "^2.6.1"
  }
}

**Comment by ammarkarachi** [comment]
Thanks for the update I tried using the same libraries and reiniting, I am suspecting there is a missing package in the node modules of you amplify installation. Would you be able to reinstall amplify-cli?

**Comment by alexlao** [comment]
I've cleared my workspace, reinstall amplify-cli and am getting the same issue. I was fine before I updated my amplify version the other day, which is why I suspect it could be a versioning issue.

Do you want to see the repo's code itself?

**Comment by ammarkarachi** [comment]
@alexlao If it's ok with you sharing that would definitely help.

**Comment by alexlao** [comment]
@ammarkarachi is there a way I can send a zip to you other than sharing publicly on GitHub?

**Comment by ammarkarachi** [comment]
I am sorry I should have asked you to email it earlier to amplify-cli@amazon.com be sure to put the issue number in the subject

**Comment by alexlao** [comment]
Thanks. Just sent. Gmail blocks tars from sending, so I've attached a link that points to my google drive to access the tar

**Comment by ammarkarachi** [comment]
@alexlao Thanks!

**Comment by sundarcodes** [comment]
Had the same issue. The issue we had was because of broken symbolic links inside one of then node_modules/.bin folder. The symbolic links were having a backslash instead of forward slash. 
![Screenshot 2020-10-02 at 5 33 36 PM](https://user-images.githubusercontent.com/10928793/94986959-8fb6b280-0580-11eb-88f6-a4006cd54fa7.png)

We resorted to downloading the #current-cloud-backend.zip from the S3 bucket which has the last pushed changes. Fixed up the broken sym links and re-uploaded to the S3 bucket again. `amplify pull` worked then !!

Not sure if this was the right way to go but just sharing what worked for us or rather how me made it work. Would love to know if we have better solutions for this..
